content/sales.rst

```diff
@@ -0,0 +1,152 @@
+Sales and Marketing
+###################
+
+:menu_order: 008
+
+.. contents::
+
+
+Introduction
+============
+
+This page briefly describes operational Sales details.
+
+
+Product Demo
+============
+
+Please see the `DemoSession` wiki page for the demo session structure.
+
+
+Demo text template to use
+-------------------------
+
+The demo session content is similar to the documentation in that it is aimed at
+system administrators and other individuals involved.
+
+Please send your criteria of requirements so that we can
+better answer your questions in advance of the demo.
+
+For those involved in procurement, please contact the Sales team.
+
+**Pre-demo session:**
+
+We recommend obtaining the trial version and installing until the
+`initial stage <https://www.sftpplus.com/documentation/sftpplus/latest/getting-started.html>`_
+
+In that way, you are able to continue after the demo session.
+
+
+Items to consider agreeing to a demo:
+-------------------------------------
+
+Is the customer actively pursuing a trial or considering a trial?
+
+If they are still considering a trial, it would be best to get a list of
+requirements first, and send through response.
```

Comment:
isn't this a repetition of what is already contained in the demo template?

The tempalate servers 2 purposes , it is a simply copy/paste source, but it also documents our process ... so we don't need to re-document outside of the template